responses.py

```diff
@@ -0,0 +1,35 @@
+
+#please add/edit to this.
+mods =['Astronuts', 'Nyxibae', 'bk, m8']
```

Comment:
You're missing hippo, Andrew, vim and begin

Reply:
I'll get an update going! :D